Dockerfile

```diff
@@ -38,8 +44,8 @@ ARG POSTGRESQL_VERSION=11.14
 #
 # We also set up a `rust` user by default. This user has sudo privileges if you
 # need to install any more software.
-RUN apt-get update && \
-    export DEBIAN_FRONTEND=noninteractive && \
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
```

Comment:
Please leave out any minor changes that don't change anything, unless the current version is clearly inelegant.